Containerfile

```diff
@@ -0,0 +1,21 @@
+FROM centos:stream8
+
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+
+RUN echo "[git-annex]" >> /etc/yum.repos.d/git-annex.repo \
+         && echo "name=git-annex" >> /etc/yum.repos.d/git-annex.repo \
+         && echo "baseurl='https://downloads.kitenet.net/git-annex/linux/current/rpms/'" >> /etc/yum.repos.d/git-annex.repo \
+         && echo "gpgcheck=0" >> /etc/yum.repos.d/git-annex.repo \
+         && echo "enabled=1" >> /etc/yum.repos.d/git-annex.repo
+
+RUN dnf -y module enable nodejs:12 ruby:3.0
+RUN dnf -y install nodejs vim git rpmdevtools git-annex-standalone wget python3 ruby jq ruby-devel make gcc-c++ postgresql-devel libvirt-devel libxml2-devel libcurl-devel systemd-devel
+
+RUN npm install npm2rpm --global
+
+RUN pip3 install packaging requests
```

Comment:
Shouldn't this install obal and the rest come via deps?

Reply:
We can install Obal. This is actually required by one of the scripts in foreman-packaging.